app/scenes/channel/channel_dropdown/channel_dropdown_container.js

```diff
@@ -0,0 +1,3 @@
+import ChannelDropdown from './channel_dropdown';
```

Comment:
@jarredwitt Looks like we're missing a copyright colophon here.

@enahum, @hmhealey I doubt that we'll ever stop forgetting to include this until we start auto inserting it as part of our workflow.